styles/android/spark_design_tokens_dimens.xml

```diff
@@ -174,9 +174,10 @@
   <dimen name="sprk_masthead_logo_max_width">192.00dp</dimen><!-- The maximum width of the logo in the Masthead. -->
   <dimen name="sprk_masthead_logo_min_width">174.00dp</dimen><!-- The minimum width of the logo in the Masthead. -->
   <dimen name="sprk_masthead_selector_padding">8.00dp</dimen><!-- The padding of the selector in the Masthead with Extended Navigation. -->
-  <dimen name="sprk_masthead_selector_dropdown_padding">0.00dp</dimen><!-- The padding of the selector dropdown in the Masthead with Extended Navigation. -->
   <dimen name="sprk_masthead_selector_min_width">272.00dp</dimen><!-- The minimum width of the selector in the Masthead with Extended Navigation. -->
   <dimen name="sprk_masthead_selector_border_radius">5.00dp</dimen><!-- The border radius of the selector in the Masthead with Extended Navigation. -->
+  <dimen name="sprk_masthead_selector_dropdown_padding">0.00dp</dimen><!-- The padding of the Masthead Selector Dropdown. -->
```

Comment:
Why'd you update the comment to remove the part about it being for the Extended variant? It's useful because it won't apply to the default variant.

Reply:
I updated all token descriptions to make sure that it's mentioning extended navigation!

Reply:
Nice, thank you! It looks like those updates didnt come through here. Did you commit the regenerated file?